src/modules/common/environment/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+const SUPABASE_URL = process.env.SUPABASE_URL || '';
+const SUPABASE_KEY = process.env.SUPABASE_KEY || '';
+const SUPABASE_JWT_SECRET = process.env.SUPABASE_JWT_SECRET || '';
+const ENABLE_CATEGORY_MUTATION = process.env.ENABLE_CATEGORY_MUTATION === 'true';
+
+export {
```

Comment:
@PCalape You can export variables using `export const VARIABLE_NAME = '...';`, is there a reason for exporting them like this?

Reply:
I think your suggestion is better @arnellebalane. No particular reason in my current implementation thanks!